.github/actions/run_awx_devel/action.yml

```diff
@@ -0,0 +1,92 @@
+# This currently *always* uses the "warm build cache" image
+# We should do something to allow forcing a rebuild, probably by looking for
+# some string in the commit message or something.
+
+name: Run AWX (devel environment)
+description: Runs AWX with `make docker-compose`
+inputs:
+  github-token:
+    description: GitHub Token for registry access
+    required: true
+  build-ui:
+    description: Should the UI be built?
+    required: false
+    default: false
+    type: boolean
+outputs:
+  ip:
+    description: The IP of the tools_awx_1 container
+    value: ${{ steps.data.outputs.ip }}
+  admin-token:
+    description: OAuth token for admin user
+    value: ${{ steps.data.outputs.admin_token }}
+runs:
+  using: composite
+  steps:
+    - name: Get python version from Makefile
+      shell: bash
+      run: echo py_version=`make PYTHON_VERSION` >> $GITHUB_ENV
+
+    - name: Remove built-in ansible-core
+      shell: bash
+      run: /usr/bin/python3 -m pip uninstall -y ansible-core
```

Comment:
I'm getting a little confused here.

Reply:
I can add a comment. The idea is: the GHA runner actually ships with ansible-core installed against the default Python. This removes it so we can install a new python (with setup-python) and latest ansible-core.

Reply:
makes sense, although I didn't have a particular reason to question the stock python they have